rcon/game_logs.py

```diff
@@ -312,46 +319,69 @@ def get_recent_logs(
     min_timestamp=None,
     exact_player_match=False,
     exact_action=False,
+    actions_filter_in=True,
```

Comment:
Why not calling this one "inclusive_filter' it's not great but maybe a bit more intutive than the current one?

Reply:
That would be much more clear! Changes made.